nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -487,6 +522,20 @@ def forward(self, audio_signal, length, cache_last_channel=None, cache_last_time
                 _, pos_emb = self.pos_enc(x=audio_signal, cache_len=cache_len)
                 pad_mask, att_mask = self._create_masks(max_audio_length, length, audio_signal.device)
 
+            # saving tensors if required for interctc loss
+            if self.is_access_enabled():
```

Comment:
If self.is_access_enabled() and self.is_interctc_enabled():

Reply:
So, this function is defined on the EncDec model level, not in the encoder. Here we can only check that by querying access mixin and checking if it needs to log something. Do you propose to add a new is_interctc_enabled function to all encoders and put that mixin access check there?

Reply:
Hmm. The glob access mixin config can be used to see whether interctc is enabled of disabled (we can set a config value when you call set_interctc_available() in the mixin. 

So basically if is_access_enabled(): if self.access_cfg.get("is_interctc_enabled", False): ...